packages/commons-ui-core/src/NavBar/NavBar.stories.js

```diff
@@ -0,0 +1,49 @@
+import Box from "@mui/material/Box";
+import React from "react";
+
+import NavBar from "./NavBar";
```

Comment:
from `"."` ?

Reply:
Yeah, it would work I think. I just want to be explicit here that `NavBar.{stories,test}.js` are for components in `NavBar.js` file since in some components, a single folder can contain multiple related [components](https://github.com/mui/material-ui/tree/master/packages/mui-material/src/ButtonBase) @gertie-sheshe